src/spectrecoff/Spectre/Status.fs

```diff
@@ -53,3 +45,59 @@ let updateWithCustomSpinner spinner (context: StatusContext) =
         | None -> context.SpinnerStyle
 
     context
+
+let start statusText (operation: StatusOperation) =
+    task {
+        return! AnsiConsole
+            .Status()
+            .StartAsync(statusText, operation)
+    }
+
+let startWithResult<'Result> statusText (operation: StatusOperation<'Result>) =
+    task {
+        return! AnsiConsole
+            .Status()
+            .StartAsync(statusText, operation)
+    }
+
+let startWithCustomSpinner spinner (operation: StatusOperation) =
```

Comment:
As said above, I think we should not support two ways, rather use the generic and return `unit`.

Reply:
Done